src/Cake.Common/Tools/VSWhere/Product/VSWhereProduct.cs

```diff
@@ -55,6 +56,10 @@ private ProcessArgumentBuilder GetArguments(VSWhereProductSettings settings)
             {
                 builder.AppendQuoted(settings.Products);
             }
+            else
+            {
+                builder.AppendQuoted("*");
```

Comment:
I also get the same error with the newest version.

![image](https://cloud.githubusercontent.com/assets/1568133/24018981/8dd4eef4-0a6c-11e7-977d-b749510c2d47.png)